tortoise/sim/generator.go

```diff
@@ -247,3 +247,8 @@ func (g *Generator) generateAtxs() {
 		}
 	}
 }
+
+// Layer returns generated layer.
+func (g *Generator) Layer(i int) *types.Layer {
+	return g.layers[i+1]
```

Comment:
is it possible we try to access non-existing layer and then it panics?

Reply:
this is used exclusively for testing
it surely will panic if non-existing layers is requested